website/client/components/groups/group.vue

```diff
@@ -552,6 +552,30 @@ export default {
       this.$store.state.memberModalOptions.fetchMoreMembers = this.loadMembers;
       this.$root.$emit('bv::show::modal', 'members-modal');
     },
+    disableMessageSendShortcut () {
+      // Some users were experiencing accidental sending of messages after pasting
+      // So, after pasting, disable the shortcut for a second.
+      this.chatSubmitDisable = true;
+
+      if (this.chatSubmitTimeout) {
+        // If someone pastes during the disabled period, prevent early re-enable
+        clearTimeout(this.chatSubmitTimeout);
+        this.chatSubmitTimeout = null;
+      }
+
+      let This = this;
+
+      this.chatSubmitTimeout = window.setTimeout(() => {
+        This.chatSubmitTimeout = null;
```

Comment:
there shoudn't be a need to use temp variables for `this` during compilation everything inside a `lamdba method` can access the outer scope